tls/extensions/s2n_server_session_ticket.c

```diff
@@ -20,29 +20,53 @@
 #include "tls/s2n_tls.h"
 #include "tls/extensions/s2n_server_session_ticket.h"
 
-#define s2n_server_can_send_nst(conn) (s2n_server_sending_nst((conn)) && \
-        (conn)->actual_protocol_version < S2N_TLS13)
+static bool s2n_session_ticket_should_send(struct s2n_connection *conn);
+static int s2n_session_ticket_send(struct s2n_connection *conn, struct s2n_stuffer *out);
+static int s2n_session_ticket_recv(struct s2n_connection *conn, struct s2n_stuffer *extension);
 
-int s2n_recv_server_session_ticket_ext(struct s2n_connection *conn, struct s2n_stuffer *extension)
+const s2n_extension_type s2n_server_session_ticket_extension = {
+    .iana_value = TLS_EXTENSION_SESSION_TICKET,
+    .is_response = true,
+    .send = s2n_session_ticket_send,
+    .recv = s2n_session_ticket_recv,
+    .should_send = s2n_session_ticket_should_send,
+    .if_missing = s2n_extension_noop_if_missing,
+};
+
+static bool s2n_session_ticket_should_send(struct s2n_connection *conn)
+{
+    return s2n_server_sending_nst(conn) && s2n_connection_get_protocol_version(conn) < S2N_TLS13;
+}
+
+int s2n_session_ticket_send(struct s2n_connection *conn, struct s2n_stuffer *out)
+{
+    /* Write nothing. The extension just needs to exist. */
+    return S2N_SUCCESS;
+}
+
+int s2n_session_ticket_recv(struct s2n_connection *conn, struct s2n_stuffer *extension)
```

Comment:
`static` missing here and `extension` is unused within this function

Reply:
`extension` is required to match the function pointer prototype.